sdk/storage/azure-storage-blob/CHANGELOG.md

```diff
@@ -15,10 +37,16 @@ For details on the Azure SDK for Java (August 2019 Preview) release refer to the
 - Removed `AnonymousClientCredential` class.
 - Changed parameter ordering of `BlobClient` and `BlobAsyncClient` `download` and `downloadToFile`.
 
-## Version 12.0.0-preview.1:
+This package's
+[documentation](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-blob_12.0.0-preview.2/sdk/storage/azure-storage-blob/README.md)
+and
+[samples](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-blob_12.0.0-preview.2/sdk/storage/azure-storage-blob/src/samples/java/com/azure/storage/blob)
+demonstrate the new API.
+
+## Version 12.0.0-preview.1 (2019-06-28):
+Version 12.0.0-preview.1 is a preview of our efforts in creating a client library that is developer-friendly, idiomatic to the Java ecosystem, and as consistent across different languages and platforms as possible. The principles that guide our efforts can be found in the [Azure SDK Design Guidelines for Java](https://azuresdkspecs.z5.web.core.windows.net/JavaSpec.html).
```

Comment:
Should this link be https://azure.github.io/azure-sdk/java_introduction.html ?

Reply:
Fixed